packages/components/src/components/toggle-button/toggle-button.css

```diff
@@ -0,0 +1,191 @@
+/**
+ * @license
+ * Scale https://github.com/telekom/scale
+ *
+ * Copyright (c) 2021 Egor Kirpichev and contributors, Deutsche Telekom AG
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+:host {
+  --width: auto;
+  --spacing-x: var(--scl-spacing-24);
+  --spacing-x-icon-only: var(--scl-spacing-8);
+  --min-height: var(--scl-spacing-24);
+  --xs-height: var(--scl-spacing-24);
+  --small-height: var(--scl-spacing-32);
+  --regular-height: var(--scl-spacing-40);
+  --large-height: var(--scl-spacing-48);
+  --radius: var(--scl-radius-8);
+  --transition: all var(--scl-motion-duration-fast)
+    var(--scl-motion-easing-standard);
+  --box-shadow-focus: inset 0 0 0 var(--scl-spacing-2) var(--scl-color-focus);
+  --font-weight: var(--scl-font-weight-bold);
+  --font-size-large: var(--scl-font-size-16);
+  --font-size-small: var(--scl-font-size-14);
+  --font-size-xs: var(--scl-font-size-12);
+  --line-height: var(--scl-font-line-height-150);
+  --spacing-icon-x: var(--scl-spacing-8);
+  --vertical-align: middle;
+
+  /* size: small */
+  --font-size-small: var(--scl-font-size-12);
+  --line-height-small: var(--scl-font-line-height-133);
+  --min-height-small: var(--scl-spacing-32);
+
+  /* variant: primary */
+  --radius-primary: var(--radius);
+  --background-primary: #dcdcdc;
+  --background-primary-disabled: #f4f4f4;
+  --color-primary: #262626;
+  --color-primary-hover: var(--scl-color-primary-hover);
+  --color-primary-active: var(--scl-color-primary-active);
+  --color-primary-disabled: #d0d0d0;
+  --color-primary-background-disabled: #f4f4f4;
+
+  /* variant: secondary different from primary*/
+  --background-secondary: var(--scl-color-white);
+
+  /* selected */
+  --color-selected: var(--scl-color-white);
+  --color-background-selected-magenta: var(--scl-color-primary);
+  --color-background-selected-hover-magenta: var(--color-primary-hover);
+  --color-background-selected-active-magenta: var(--scl-color-primary-active);
+  --color-background-selected-black: #262626;
+  --color-background-selected-hover-black: #726c6c;
+  --color-background-selected-active-black: #000000;
+}
+
+.toggle-button {
+  box-sizing: border-box;
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  border: 0;
+  outline: none;
+  cursor: pointer;
+  user-select: none;
+  font-family: inherit;
+  word-spacing: inherit;
+  letter-spacing: inherit;
+  justify-content: center;
+  text-decoration: none;
+  font-weight: var(--font-weight);
+  font-size: var(--font-size);
+  line-height: var(--line-height);
+  min-height: var(--min-height);
+  width: var(--width);
+  padding-left: var(--spacing-x);
+  padding-right: var(--spacing-x);
+  vertical-align: var(--vertical-align);
+  transition: var(--transition);
+  background: #fff;
```

Comment:
do you really need a white background?

Reply:
no need for this background. I have deleted it ;-)